js.md

```diff
@@ -102,7 +133,7 @@ hooks](#html-class-hooks).
 
 ## Strict mode
 
-You should add the `"use strict";` statement to the top of your module functions.
+You should add the `"use strict"` statement to the top of your module functions.
```

Comment:
Had a hard time going through everything since it's not a real codebase, thanks for the keen eye 👍